zeeguu/api/endpoints/accounts.py

```diff
@@ -13,6 +13,25 @@
 from zeeguu.logging import log
 
 
+@api.route("/delete_user/<session_id>", methods=["POST"])
+@cross_domain
+def remove_user(session_id):
+    from zeeguu.core.account_management.user_account_deletion import delete_user_account
+
+    try:
+        session_id = int(session_id)
```

Comment:
why do we pass the session_id in the URL? Can't we use the @with_session decorator?

Reply:
That's a good point -  I will check if it works using the decorator!